src/components/Form/Form.js

```diff
@@ -9,9 +10,11 @@ import { Label } from '../UI/Form/Label.styled';
 import { Button } from '../UI/Button.styled';
 import { Error } from '../UI/Form/Error.styled';
 
-export default function Form({ onAddEntry, modalShow }) {
+export default function Form() {
+	const addEntry = useStore(state => state.addEntry);
+	const modalShow = useStore(state => state.modalShow);
 	const onSubmit = (data, event) => {
```

Comment:
Maybe you will need:
event.preventDefault()
someday ;)

Reply:
As discussed I don't need this when using React Hook Form.